src/metabase/pulse/render/body.clj

```diff
@@ -601,66 +567,99 @@
-                                   group-key)
-            card-color         (or (series-setting viz-settings group-key :color)
-                                   (nth colors idx))
+            column-name          (or (series-setting viz-settings group-key :name)
+                                   (series-setting viz-settings group-key :title))
             card-type          (or (series-setting viz-settings group-key :display)
                                    chart-type
                                    (nth default-combo-chart-types idx))
             y-axis-pos         (or (series-setting viz-settings group-key :axis)
                                    (nth (default-y-pos viz-settings) idx))]
-        {:name          card-name
-         :color         card-color
+        {:name          column-name
+         :cardName      card-name
          :type          card-type
          :data          selected-row-group
-         :yAxisPosition y-axis-pos}))))
+         :yAxisPosition y-axis-pos
+         :column        (second x-cols)
+         :breakoutValue group-key}))))
+
+(defn- render-multiple-lab-chart
+  "When multiple non-scalar cards are combined, render them as a line, area, or bar chart"
+  [render-type card dashcard {:keys [viz-settings]
+                              :as   data}]
+  (let [viz-settings      (merge viz-settings (:visualization_settings dashcard))
+        multi-res         (pu/execute-multi-card card dashcard)
+        ;; multi-res gets the other results from the set of multis.
+        ;; we shove cards and data here all together below for uniformity's sake
+        viz-settings      (set-default-stacked viz-settings card)
+        viz-settings-seqs (cons viz-settings (map (comp :visualization_settings :card) multi-res))
+        cards             (cons card (map :card multi-res))
+        multi-data        (cons data (map #(get-in % [:result :data]) multi-res))
+        axis-row-fns      (fn [card data]
+                            [(or (ui-logic/mult-x-axis-rowfn card data) #(vector (first %)))
+                             (or (ui-logic/mult-y-axis-rowfn card data) #(vector (second %)))])
+        rowfns            (mapv axis-row-fns cards multi-data)
+        row-seqs          (for [[row-seq [x-rowfn y-rowfn]] (map vector (map :rows multi-data) rowfns)]
+                            (map (comp flatten (juxt x-rowfn y-rowfn))
+                                 (common/row-preprocess x-rowfn y-rowfn row-seq)))
+        col-seqs          (map :cols multi-data)
+        first-rowfns      (first rowfns)
+        [[x-col] [y-col]] ((juxt (first first-rowfns) (second first-rowfns)) (first col-seqs))
+        labels            (x-and-y-axis-label-info x-col y-col viz-settings)
+        names             (map :name cards)
+        types             (replace {:scalar :bar} (map :display cards))
+        settings-seqs     (map (partial ->ts-viz x-col y-col labels) viz-settings-seqs)
```

Comment:
This isn't used on FE yet. I initially thought it was used, but [after asking in Slack](https://metaboat.slack.com/archives/C01LQQ2UW03/p1668575980848439), it turns out we should aim to achieve feature parity so only the first settings (the settings of the original question of the dashcard) is used.